src/main/java/com/smartbear/readyapi/client/assertions/ValidHttpStatusCodesAssertionBuilder.java

```diff
@@ -9,21 +9,30 @@
 import static com.smartbear.readyapi.client.Validator.validateNotEmpty;
 
 public class ValidHttpStatusCodesAssertionBuilder<T extends Assertion> extends AbstractAssertionBuilder<Assertion> implements HttpStatusCodeAssertionBuilder {
-    protected List<Integer> statusCodes = new ArrayList<>();
+    protected List<String> statusCodes = new ArrayList<>();
     protected String name;
 
     @Override
     public ValidHttpStatusCodesAssertionBuilder addStatusCode(int statusCode) {
```

Comment:
in line with the above comment - an addStatusCode( String statusCode ) method...